### ISSUE TITLE
host.scheduleTask very slow on Windows 10

### ISSUE THREAD

**Issue by git-moss** [teotigraphix/Framework4Bitwig#261]
Since one of the 3.2.x versions the schaduleTask method is very slow on Windows 10. I could not reproduce that on the other OSs.
E.g. a 400 ms timeout takes about 3 seconds to fire.

**Comment by alexr1221** [comment]
Hello,

I also have this problem, it adds ~20% of time in my case. I checked with Date.now() before and after.

**Comment by git-moss** [comment]
In my case it was a side effect of the slow USB sending (the Push 2 display process). So not actually a bug.
You should check if you have processing happening the flush method which takes up some time. 20% is not that surprising.

**Comment by alexr1221** [comment]
Thanks for the reply. I had saw your other post on Bitwig's forum, I checked what could slow down the process but the only thing I use is noteInput.sendRawMidiEvent(). 
The thing that is curious is it's not just 20% added to the initial delay but 20% multiplied to the initial delay. Just like if the system's clock was not well setup, with 1sec = 1.2sec. 
I got around this problem by multiplying the delay by a determined number.
